src/app/components/translation/translation.component.ts

```diff
@@ -16,7 +16,13 @@ export class TranslationComponent {
   getPages(): AbstractPage[] {
     return this.translation.resource.pages.map(page => {
       const customPage: CustomPage = this.translation.language['custom-pages'].find(c => c.page.id === page.id);
-      return customPage == null ? page : customPage;
+
+      if (customPage == null) {
```

Comment:
Could do `if (!customPage) {`. `null` is a [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value. Not a big deal as long as it works.

Generally it seems that using `==` is discouraged. From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#Loose_equality_using:

> Some developers consider that it is pretty much never a good idea to use loose equality. The result of a comparison using strict equality is easier to predict, and as no type coercion takes place the evaluation may be faster.